go-controller/vendor/github.com/coreos/go-iptables/iptables/iptables.go

```diff
@@ -491,6 +578,57 @@ func (ipt *IPTables) run(args ...string) error {
 	return ipt.runWithOutput(args, nil)
 }
 
+// runWithOutput runs an iptables command with the given arguments,
+// writing any stdout output to the given writer
+func (ipt *IPTables) runRestore(args []string, input string) error {
```

Comment:
this seems very similar to runWithOutput for the future maybe we can call runWithOutput from within runRestore?

Reply:
sure, but in the future we will remove using this library for an nftables one so this is just temporary